IBuildaAZoo/IBuildaAZoo/Dog.cs

```diff
@@ -3,8 +3,12 @@ namespace IBuildaAZoo
 {
     public class Dog
     {
-        public Dog()
+        public abstract class Dog()
```

Comment:
You might actually want `public abstract class Dog` so you can specify some specific dogs have `Size = "Medium"` and others have `Size = "Small"`.